cmd/swapcli/main.go

```diff
@@ -19,29 +19,32 @@ import (
 	"github.com/athanorlabs/atomic-swap/common"
 	"github.com/athanorlabs/atomic-swap/common/rpctypes"
 	"github.com/athanorlabs/atomic-swap/common/types"
+	"github.com/athanorlabs/atomic-swap/relayer"
 	"github.com/athanorlabs/atomic-swap/rpcclient"
 	"github.com/athanorlabs/atomic-swap/rpcclient/wsclient"
 )
 
 const (
 	defaultDiscoverSearchTimeSecs = 12
 
-	flagSwapdPort       = "swapd-port"
-	flagMinAmount       = "min-amount"
-	flagMaxAmount       = "max-amount"
-	flagPeerID          = "peer-id"
-	flagOfferID         = "offer-id"
-	flagOfferIDs        = "offer-ids"
-	flagExchangeRate    = "exchange-rate"
-	flagProvides        = "provides"
-	flagProvidesAmount  = "provides-amount"
-	flagRelayerFee      = "relayer-fee"
-	flagRelayerEndpoint = "relayer-endpoint"
-	flagSearchTime      = "search-time"
-	flagDetached        = "detached"
+	flagSwapdPort      = "swapd-port"
+	flagMinAmount      = "min-amount"
+	flagMaxAmount      = "max-amount"
+	flagPeerID         = "peer-id"
+	flagOfferID        = "offer-id"
+	flagOfferIDs       = "offer-ids"
+	flagExchangeRate   = "exchange-rate"
+	flagProvides       = "provides"
+	flagProvidesAmount = "provides-amount"
+	flagRelayerFee     = "relayer-fee"
```

Comment:
if there's only one relayer-fee flag, how do we differentiate between the fee we want to pay for a relayed tx vs the fee we want to receive as a relayer?